example/app.json

```diff
@@ -6,8 +6,14 @@
       "displayName": "App"
     }
   },
-  "resources": [
-    "dist/assets",
-    "dist/main.jsbundle"
-  ]
+  "resources": {
```

Comment:
I think we should be able to support the flattened structure for platform specific packages. E.g. for Android only packges:
```json
"resources": {
  "bundle": "dist/main.jsbundle",
  "resources": "dist/res"
}
```

Reply:
Is that a good practice though? I think it is fine just to have ios or android objects if you want to consume the test app only for one platform.

Reply:
On top of that, it will reduce complexity on our side.

Reply:
I think it's a nice-to-have. Not sure how it would add complexity though. If the `android` key exists, use that; otherwise use `resources`. I don't think that would be too hard to add.

I would also like to get rid of the keys here and go with a flat structure (see comment below).

Reply:
Right, but iOS for example, expects an array or strings, while android expects an object. I am just not comfortable with the idea of encoding these implicit rules into the manifest, rather than having it a little bit more "boilerplaty" and consistent.